### PR TITLE
Add infer & test scripts for baidu_en8k model

### LINKED ISSUE #53: Add infer & test scripts for baidu_en8k model

### PR DESCRIPTION
**Pull request by kuke**
Resolve #53

### CODE REVIEW

File: examples/baidu_en8k/run_test_golden.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bash
+
+cd ../.. > /dev/null
+
+# download language model
+cd models/lm > /dev/null
+#sh download_lm_en.sh
```
Comment (pkuyym) [inline examples/baidu_en8k/run_test_golden.sh]: Why comment here？

File: examples/baidu_en8k/run_test_golden.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+#if [ $? -ne 0 ]; then
+#    exit 1
+#fi
+cd - > /dev/null
+
+
+# download well-trained model
+cd models/baidu_en8k > /dev/null
+#sh download_model.sh
```
Comment (pkuyym) [inline examples/baidu_en8k/run_test_golden.sh]: Please keep consistent with other example.

File: examples/baidu_en8k/run_test_golden.sh (hunk 1 above)
Comment (kuke) [reply examples/baidu_en8k/run_test_golden.sh]: A mistake, modified.

File: examples/baidu_en8k/run_test_golden.sh (hunk 2 above)
Comment (kuke) [reply examples/baidu_en8k/run_test_golden.sh]: Done

**Pull request merged**